uberenv.py

```diff
@@ -774,17 +792,25 @@ def show_info(self):
     def install(self):
         # use the uberenv package to trigger the right builds
         # and build an host-config.cmake file
-
         if not self.use_install:
             install_cmd = "spack/bin/spack "
             if self.opts["ignore_ssl_errors"]:
                 install_cmd += "-k "
             if not self.opts["install"]:
-                install_cmd += "dev-build --quiet -d {0} ".format(self.pkg_src_dir)
+                if self.uberenv_package_name:
+                    # fake package path
```

Comment:
This isnt always a fake package.  This would also be helpful for installing an actual package that exists. Or am i missing something?

Like if i was in serac and wanted to build axom in their setup.

Reply:
if `uberenv_package_name`  exists, it doesn't use the dev build logic. that pretty much it

Reply:
I added `self.use_dev_build` as a better way of identifying the logic